client/src/main/java/tw/waterball/survive/io/client/sprite/weapon/GunSprite.java

```diff
@@ -1,7 +1,7 @@
 package tw.waterball.survive.io.client.sprite.weapon;
 
 public class GunSprite extends WeaponSprite {
-	private MagazineSprite magazineSprite;
+	private tw.waterball.survive.io.client.sprite.weapon.MagazineSprite magazineSprite;
```

Comment:
import tw.waterball.survive.io.client.sprite.weapon.MagazineSprite;

Reply:
.. I don't think @Wally5077 mean this.